frontend/src/components/Graph/yAxisConfig.ts

```diff
@@ -48,8 +43,33 @@ export const getYAxisFormattedValue = (
 
 export const getToolTipValue = (value: string, format: string): string => {
 	try {
+		let decimalPrecision: number | undefined;
```

Comment:
Can we add the logic used here as comment, the context would be lost in the future and it will be difficult to change without proper documentation of the logic.

Reply:
Sure.